lib/modules/datasource/helm/index.ts

```diff
@@ -63,9 +63,9 @@ export class HelmDatasource extends Datasource {
       }
       const result: HelmRepositoryData = {};
       for (const [name, releases] of Object.entries(doc.entries)) {
-        const latestRelease = releases.sort((r0, r1) =>
-          gt(r0.version, r1.version) ? -1 : 1
-        )[0];
+        const [latestRelease] = releases.sort((r0, r1) =>
+          semverCoerced.api.isGreaterThan(r0.version, r1.version) ? -1 : 1
```

Comment:
we do have a sort function on our versioning which should be used instead 😉

Reply:
It only sorts versions but here we’re working on releases with a version property